flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormatTest.java

```diff
@@ -233,6 +255,30 @@ public void testFlush() throws SQLException, IOException {
 		}
 	}
 
+	@Test
+	public void testMetricsSetup() throws IOException {
```

Comment:
Can you extend this test to check that the metrics are correctly set (except for the `durationMs` histogram)?